src/mapeo-manager.js

```diff
@@ -442,9 +443,10 @@ export class MapeoManager extends TypedEmitter {
 
     // 7. Load config, if relevant
     // TODO: see how to expose warnings to frontend
-    /* eslint-disable no-unused-vars */
```

Comment:
Changed this to two `next-line` disables. The old code disabled the `no-unused-vars` rule _for the rest of the file_, which was probably not the intent.

Not strictly related to this change, though.